src/web/tools/NodeManager.ts

```diff
@@ -21,7 +21,9 @@ export default abstract class NodeManager {
     handler: InstanceType<ValueOf<typeof Gestures>>
   ): void {
     if (handlerTag in this.gestures) {
-      throw new Error(`Handler with tag ${handlerTag} already exists`);
+      throw new Error(
+        `Handler with tag ${handlerTag} already exists. Please ensure that no GestureHandler instance is used across multiple GestureDetectors.`
```

Comment:
```suggestion
        `Handler with tag ${handlerTag} already exists. Please ensure that no Gesture instance is used across multiple GestureDetectors.`
```

Reply:
Done in [0b178e3](https://github.com/software-mansion/react-native-gesture-handler/pull/2694/commits/0b178e350481ae074caccd46657da242d2f48f65)